compare_template_text/normalize_template_text.py

```diff
@@ -53,13 +55,44 @@ def remove_bulletted_text(self):
         
         self.template_string = re.sub('(?<=\<\<var;name\=\*bullet\*).*?(?=\>\>)','',
                                    self.template_string)
+        self.template_string = re.sub('\<\<var;name=\*bullet.*?\>\>','',self.template_string)
+        self.remove_repeating_chars()
+        return
+    
+    def remove_replaceable_text(self):
+        """The Replaceable Text is found between the <<var copyright tags. This method removes all
+        the text between the original fields from the template and the text. """
+        
+        lists = re.findall('(?<=\<\<var;name=\*;original=).*?(?=;match=.*?\>\>)',
+                           self.template_string)
+        print(lists)
+        for x in lists:
+            if(x.startswith('*')):
+                x = x[1:]
+
+            if(x.endswith('*')):
+                x = x[:-1]
+
+            self.text_string = self.text_string.replace(x,'')
```

Comment:
This has the same issue as the optional approach in that it may match and remove text that shouldn't be removed - you can just add this to the optional issue to be fixed in the future - probably taking the same solution approach